src/data/metadata.csv

```diff
@@ -9023,9 +9023,8 @@ IN_AS,IN,India,AS,Assam,,,,,,
 IN_BR,IN,India,BR,Bihar,,,,,,
 IN_CH,IN,India,CH,Chandigarh,,,,,,
 IN_CT,IN,India,CT,Chhattisgarh,,,,,,
-IN_DD,IN,India,DD,Daman and Diu,,,,,,
```

Comment:
Were we not getting data from this particular subregion?

Reply:
Re-added because mobility reports are reported for this region.